roles/logrotate/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- block:
+    - name: Install logrotate
+      package:
+        name: logrotate
+        state: present
```

Comment:
you may want to specify a version here rather than `present`. This will create inconsistencies when ran over a long period of time.